app/1.0/configuration.md

```diff
@@ -351,7 +351,7 @@ http {
     # here, we declare our custom location serving our website
     # (or API portal) which we can optimize for serving static assets
     location / {
-      root /var/www/my-api.com;
+      root /var/www/example.com;
```

Comment:
I like the prior example better - it is more clear to me.

Reply:
I'm sorry but I cannot comply with this one. www.my-api.com is a real business and we should not be linking to it in our docs. I have been burned by making a domain up from random words and it turned out to be ... not safe for work).

`example.net` is a reserved domain name that is guaranteed to be usable in documentation and manuals, so it doesn't have that problem.